docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -209,3 +209,57 @@ When I save `$attributeName` attribute value of element located `$locator` to $s
 Scenario: Save attribute value of element
 When I save `innerText` attribute value of element located `By.id(username)` to SCENARIO variable `username`
 ----
+
+=== Change context
+
+==== *_Info_*
+
+Changes the context to an element located by locator for further localization of elements
```

Comment:
`for further localization of elements` -> `for limiting area of subsequent UI interactions`